style.css

```diff
@@ -0,0 +1,110 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap');
+
+body, html {
+    background: url(images/bg-pattern-top.svg), url(images/bg-pattern-bottom.svg), hsl(185, 75%, 39%);
+    background-repeat: no-repeat;
+    background-position: right 52vw bottom 36vh, left 50vw top 52vh;
+    /* overflow: hidden; */
+    font-family: 'Kumbh Sans', sans-serif;
+    height: 100%;
+    margin:0px;
+}
+
+.container {
+    height: 100%;
+}
+
+.card {
+    background-color: white;
+    position: fixed;
+    top: 50%;
+    left: 50%;  
+    transform: translate(-50%, -50%);
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0px 0px 20px 10px hsl(185, 75%, 39%);
+    width: 325px;
+ }
+
+ .card .pattern {
+     height: 140px;
+ }
+
+.card .pattern img {
+    height: 100%;
+    width: 100%;
+    border-radius: 20px 20px 0 0;
+}
+
+.card .picture {
+    height: 150px;
+    display: flex;
+    justify-content: center;
+    align-content: space-evenly;
+}
+
+.card .picture img {
+    border-radius: 50%;
+    border: 5px solid white;
+    position: fixed;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.card .picture p {
+    margin-top: 70px;
+    text-align: center;
+    font-size: 15px;
+    line-height: 30px;
+    color: hsl(227, 10%, 46%);
+    font-weight: 400;
+    opacity: 0.8;
+}
+
+.card .picture p span {
+    font-size: 18px;
+}
+
+.card .picture p span span {
```

Comment:
Nema razloga na ovako maloj stranici da imas ovakav CSS selektor. CSS selektori treba da su u startu sto slabiji, npr. jedna klasa, pa onda ako treba da modifikujes nesto pravis jaci selektor. Ovo ti se ponavlja na vise mesta, moja sugestija je da ubuduce to pises drugacije